docs/features/customization/opening-cline-in-sidebar.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: "Opening Cline in the Sidebar"
+description: "Learn how to open Cline in the right sidebar in VS Code and Cursor"
+---
+
+## VS Code
+
+To open Cline in the right sidebar:
+
+1. Make sure your extension view is aligned vertically to the left
+2. Click the button that opens the right side view in VS Code (the one VS Code uses for GitHub Copilot)
+3. Drag the Cline icon over to the nav panel at the top of that right view
+
+<Frame>
+	<img
+		src="https://storage.googleapis.com/cline_public_images/docs/assets/vs_code_right_view.gif"
```

Comment:
do these images work without extra headers? i tried to open, and they looked like white rectangles